index.html

```diff
@@ -357,10 +370,11 @@ <h2>Contact me</h2>
           ></textarea>
 
           <button type="submit" class="btn-body-footer btn">
-            Get It Touch
+            Get it Touch
           </button>
         </form>
       </section>
     </footer>
+    <script src="/dynamic.js"></script>
```

Comment:
- [ ] The reason that your mobile menu is not displaying correctly, it's because you are not marking in which directory dynamic.js is, the correct way is     <script src=" ./dynamic.js "></script>, the  " ./ " is like a "look in the root directory, and you always need to use it if the file is in the root directory, you can found more info about this online, in the meantime, please fix it and check if you deploy page work correctly before you ask for a new code review, is your last chance.